include/fcl/profile.h

```diff
@@ -58,7 +58,7 @@
 #include <map>
 #include <string>
 #include <iostream>
-#include <boost/thread.hpp>
+#include <thread>
```

Comment:
Add `#include <mutex>` here -- compile failed on Windows. C++11 doesn't promise that mutex will be defined in thread.